spec/models/scorm_package_spec.rb

```diff
@@ -24,13 +23,14 @@ def get_manifest_entry(zip)
   end
 
   it "should convert to zip file" do
-    zip = Zip::File.new("spec/fixtures/scorm_package.zip")
-    package = Blacklight::ScormPackage.new(zip, get_manifest_entry(zip))
+    zip, manifest = get_zip_manifest("scorm_package.zip")
```

Comment:
See previous comment -- it would be better for `get_zip_manifest` to yield `zip` and `manifest` to a block, rather than returning them, in order to keep the `zip` handle open.